Controllers/All.controller.js

```diff
@@ -0,0 +1,12 @@
+const all = (req, res)=> {
+  res.status(404);
+  res.header("Content-Type", "application/json");
+  res.end(JSON.stringify({ message: "Route not found" }));
```

Comment:
You don't need the res.header or JSON.stringify. Express done that for you :)  All you need is `res.status(404).send({ message: 'Route not found' })`